app/views/static_pages/about.html.erb

```diff
@@ -0,0 +1,11 @@
+<% provide :title, t(".title")  %>
+<h1><%= t ".about" %></h1>
+<p>
+  The <a href="http://www.railstutorial.org/"><em>Ruby on Rails
```

Comment:
Dùng toàn bộ I18n cho text ở html em nhé

tag `a` của html thì chuyển sang dùng `link_to` của rails nhé